### DISCUSSION TITLE
githubにプッシュできない

### DISCUSSION THREAD

**Discussion by I-Shizu** [community/community#123224] [Programming Help]
### Body

エラーの説明：
GitHubにプッシュしようとした際、次のエラーが発生しました。「ssh: Could not resolve hostname github.com:I-Shizu: nodename nor servname provided, or not known」。何度か試みましたが、問題が解決しません。

以前に試した解決策：

ローカルリポジトリを作成しました。
README.mdファイルを作成しました。
変更をコミットしました。
mainブランチを作成しました。
リモートリポジトリのURLを追加しました。
リモートリポジトリにプッシュしようとしました。
今回試した解決策：

SSH構成を確認し、正しいキーが追加されていることを確認しました。
ssh -T git@github.comを使用してSSH接続をテストしました。
SSHエージェントに正しいキーが追加されていることを確認しました。
SSH構成ファイルを正しいホスト名で更新しました。
nslookup github.comを使用してDNSの解決を確認しました。
ファイアウォール設定を確認し、ターミナルへのアクセスが許可されていることを確認しました。
これらの努力にもかかわらず、問題は解決されません。GitHubコミュニティからの支援を得て、エラーの根本的な原因を特定し、解決するための支援を求めています。見解や提案があれば、ぜひご意見をお聞かせください。

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @I-Shizu, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
